mage_ai/orchestration/db/migrations/versions/e3593cc2191e_add_key_uuid_to_secrets.py

```diff
@@ -0,0 +1,35 @@
+"""Add key uuid to secrets
+
+Revision ID: e3593cc2191e
+Revises: dfe49d040487
+Create Date: 2023-07-14 15:53:06.580076
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e3593cc2191e'
+down_revision = 'dfe49d040487'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('secret', schema=None) as batch_op:
+        batch_op.alter_column('name', existing_type=sa.String(length=255), unique=False, nullable=False)
```

Comment:
Do you check whether the unique constraint is dropped after migration (in sqlite and postgres)?
The unique constraint is created separately https://github.com/mage-ai/mage-ai/blob/master/mage_ai/orchestration/db/migrations/versions/1f9353eddbc6_add_secrets_table.py#L29

Reply:
I did, it seemed to work this way. I tried to manually drop the constraint, but I always got an error doing it that way